monai/transforms/utility/array.py

```diff
@@ -1715,3 +1716,124 @@ def __call__(self, img: NdarrayOrTensor, meta_dict: Mapping | None = None) -> Nd
         if self._do_transform:
             img = self.filter(img)
         return img
+
+
+class ApplyTransformToPoints(InvertibleTransform, Transform):
```

Comment:
The `ApplyTransformToPoints` class could benefit from additional description of the transformation being applied and the expected input and output formats.

Reply:
Add more descriptions in the docstring.